<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firmCalculadora</class>
 <widget class="QMainWindow" name="firmCalculadora">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(10, 36, 17, 255), stop:1 rgba(255, 255, 255, 255));
}

QLabel
{
	color: rgb(48, 0, 1);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>1 Numero</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditA"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>2 Numero</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditB"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Resultado</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditC">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>138</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtoCalcular">
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Calcular</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditA</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtoCalcular</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditB</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtoCalcular</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
